pkg/entrypoint/options_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2018 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package entrypoint
+
+import (
+	"testing"
+
+	"github.com/knative/build/pkg/entrypoint/wrapper"
+)
+
+func TestOptions_Validate(t *testing.T) {
+	var testCases = []struct {
+		name        string
+		input       Options
+		expectedErr bool
+	}{
```

Comment:
style nit: collapse `{{` here (and `}}` at the end) to save vertical and horizontal space. Put `},\n{` on one line like `}, {` for the same reason.